extension/persistence/eclipselink/src/test/java/com/snowflake/polaris/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreTest.java

```diff
@@ -49,4 +65,20 @@ protected PolarisTestMetaStoreManager createPolarisTestMetaStoreManager() {
             new PolarisConfigurationStore() {},
             timeSource.withZone(ZoneId.systemDefault())));
   }
+
+  @Test
+  void throwExceptionIfConfigFileDoesNotExists() {
+    PolarisDiagnostics diagServices = new PolarisDefaultDiagServiceImpl();
+    PolarisEclipseLinkStore store = new PolarisEclipseLinkStore(diagServices);
+    new PolarisEclipseLinkMetaStoreSessionImpl(
+        store, Mockito.mock(), () -> "realm", "does not exists", "polaris-dev");
+  }
+
+  @Test
+  void ensureNotLoadingDefaultClassPersistence() {
```

Comment:
How does this test that you're not still loading the file from `/META-INF/persistence.xml`?